src/mainwindow.cpp

```diff
@@ -1648,7 +1627,7 @@ void MainWindow::loadNoteFolderListMenu() {
     int c = 0;
     Q_FOREACH(QAction* action, actions) {
             // start with the 2nd item, the first item is the menu itself
-            if (c++ > 0) {
+            if (++c > 0) {
```

Comment:
Are you sure you can do this? Value of `c` will be increased before assignment.

Reply:
ah, no. Thanks. I will revert this.

Reply:
I will undo that change.

Reply:
Ok, I'll wait a minute more. :grin:

Reply:
eeh. Don't merge this yet :grimacing: 
I am preparing some more changes :grimacing:

Reply:
Done?

Reply:
Done.